### ISSUE TITLE
Reader Detail: Crash when changing orientation continuously

### ISSUE THREAD

**Issue by kurzee** [wordpress-mobile/WordPress-iOS#6294]
### Steps to reproduce the behavior

1. Rotate the device in the same direction until it crashes, either clockwise or counter-clockwise.
2. Should crash after going from portrait > landscape > upside down > landscape.

##### Tested on [device], iOS [version], WPiOS [version]

iPhone 6 Plus simulator, iOS 10.1

cc @aerych 

```
2016-12-06 14:09:01.089 WordPress[74521:6474656] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSLayoutConstraint constant is not finite!  That's illegal.  constant:inf firstAnchor:<NSLayoutYAxisAnchor:0x610000661440 "UIStackView:0x7fe368caf8e0.top"> secondAnchor:<NSLayoutYAxisAnchor:0x60800047d900 "WordPress.WPRichContentView:0x7fe369192400'￼￼
As the rain started to...'.top">'
*** First throw call stack:
(
	0   CoreFoundation                      0x00000001164a534b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x0000000115f0d21e objc_exception_throw + 48
	2   CoreFoundation                      0x00000001164a9442 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00000001105c9e4d -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 195
	4   Foundation                          0x0000000110549353 -[NSLayoutConstraint _setSymbolicConstant:constant:] + 269
	5   Foundation                          0x0000000110549217 -[NSLayoutConstraint setConstant:] + 68
	6   WordPress                           0x000000010d98cbc2 _TFC9WordPress26ReaderDetailViewControllerP33_6894C07EBB00C79B78761FE3A823FACE21updateTextViewMarginsfT_T_ + 1010
	7   WordPress                           0x000000010d989909 _TFFC9WordPress26ReaderDetailViewController24viewWillTransitionToSizeFTVSC6CGSize25withTransitionCoordinatorPSo37UIViewControllerTransitionCoordinator__T_U_FPSo44UIViewControllerTransitionCoordinatorContext_T_ + 41
	8   WordPress                           0x000000010d989947 _TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFo_iPS___iT__ + 23
	9   WordPress                           0x000000010d99afd1 _TPA__TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFo_iPS___iT__ + 81
	10  WordPress                           0x000000010d989890 _TTRXFo_iPSo44UIViewControllerTransitionCoordinatorContext__iT__XFo_oPS___dT__ + 32
	11  WordPress                           0x000000010d9898d8 _TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFdCb_dPS___dT__ + 56
	12  UIKit                               0x0000000112c0f539 -[_UIViewControllerTransitionCoordinator _applyBlocks:releaseBlocks:] + 311
	13  UIKit                               0x0000000112c0b2f1 -[_UIViewControllerTransitionContext _runAlongsideCompletions] + 151
	14  UIKit                               0x0000000112c0afcd -[_UIViewControllerTransitionContext completeTransition:] + 118
	15  UIKit                               0x0000000112764033 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke_2.206 + 26
	16  UIKit                               0x00000001121847e4 +[UIView(Internal) _performBlockDelayingTriggeringResponderEvents:] + 188
	17  UIKit                               0x0000000112763fe4 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke.205 + 124
	18  UIKit                               0x000000011217f6aa -[UIViewAnimationBlockDelegate _didEndBlockAnimation:finished:context:] + 528
	19  UIKit                               0x0000000112152475 -[UIViewAnimationState sendDelegateAnimationDidStop:finished:] + 222
	20  UIKit                               0x00000001121529ca -[UIViewAnimationState animationDidStop:finished:] + 136
	21  QuartzCore                          0x0000000110f98990 _ZN2CA5Layer23run_animation_callbacksEPv + 316
	22  libdispatch.dylib                   0x000000011737e0cd _dispatch_client_callout + 8
	23  libdispatch.dylib                   0x000000011735e8d6 _dispatch_main_queue_callback_4CF + 406
	24  CoreFoundation                      0x00000001164694f9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	25  CoreFoundation                      0x000000011642ef8d __CFRunLoopRun + 2205
	26  CoreFoundation                      0x000000011642e494 CFRunLoopRunSpecific + 420
	27  GraphicsServices                    0x0000000118ec5a6f GSEventRunModal + 161
	28  UIKit                               0x00000001120c5964 UIApplicationMain + 159
	29  WordPress                           0x000000010d698cee main + 62
	30  libdyld.dylib                       0x00000001173ca68d start + 1
	31  ???                                 0x0000000000000001 0x0 + 1
)
```

**Comment by aerych** [comment]
This looks like https://github.com/wordpress-mobile/WordPress-iOS/pull/6274   I just now merged release/6.8 back to develop.  Can you still repro?

**Comment by kurzee** [comment]
>This looks like #6274 I just now merged release/6.8 back to develop. Can you still repro?

Resolved! My apologies for the dupe!

**Comment by aerych** [comment]
No worries!!  Thanks for all the awesome testing :)
